deploy/monitor/prometheus_rule.yaml

```diff
@@ -23,7 +23,7 @@ spec:
           description: "The UnifiedPush Operator has been down for more than 5 minutes. "
           summary: "The UnifiedPush Operator is down. For more information see on the UnifiedPush Operator https://github.com/aerogear/unifiedpush-operator"
           sop_url: "https://github.com/aerogear/unifiedpush-operator/blob/0.1.0/SOP/SOP-operator.adoc"
-      - alert: UnifiedpushPodcount
+      - alert: UnifiedPushPodcount
         annotations:
           description: "The Pod count for the UnifiedPush has changed in the last 5 minutes."
           summary: "Pod count for namespace UnifiedPush is {{ printf '%.0f' $value }}. Expected 3 pods. For more information see on the UnifiedPush Operator https://github.com/aerogear/unifiedpush-operator"
```

Comment:
why only 3 pods? what happens if we want to scale UPS by increase the number of UPS pods?

Reply:
1. Is really possible to scale the Pods? Is the setup of the database prepared for it? 
2. How to change the qt of the pods? I am not able to do it in OCP to test/check? See here: https://master.london-4bc4.openshiftworkshop.com/console/project/unifiedpush/overview

Reply:
UPS can't be scaled right now until the work that Summers is doing is merged, so it's not exposed through the CRD yet. I don't know what you mean by "qt of the pods".

Reply:
@wei-lee so it is not possible to be scale ... 
@grdryn I do not know where you check `qt of the pods`, however, it means "quantity of the pods". See here: 

<img width="729" alt="Screenshot 2019-07-12 at 05 53 54" src="https://user-images.githubusercontent.com/7708031/61115733-79977080-a469-11e9-8b73-729e14f4efce.png">

Reply:
Where I saw "qt of the pods" was just in your comment above :) I'd suggest to always write "quantity" if that's what you mean, because "qt" could be misinterpreted as "quality" or something else.